packages/builder/src/builder.js

```diff
@@ -48,6 +48,23 @@ export default class Builder {
     this.relativeToBuild = (...args) =>
       relativeTo(this.options.buildDir, ...args)
 
+    this.serializeHead = (obj) => {
```

Comment:
This function can be potentially moved to `@nuxt/common` utils, with the enhancenment of getting `"head"` as the second param or generalizing regex. There are more use cases than would benefit from a consistent utility.

Reply:
@pi0 sure thing -- please wait until I do this before merging then :)

Reply:
@pi0 done!